source/proximity_list/proximitylist_api.cpp

```diff
@@ -77,12 +78,19 @@ static void make_pb(navitia::PbCreator& pb_creator,
             default:
                 break;
         }
+        // add stop points nearby into response
+        if (!stop_points_nearby_idx.empty()) {
+            for (const auto& sp_idx : stop_points_nearby_idx) {
+                pbnavitia::StopPoint* sp = place->add_stop_points_nearby();
+                pb_creator.fill(data.pt_data->stop_points[sp_idx], sp, depth);
```

Comment:
we also need the distance between the stop_point and the P+R to be able to calculate the crowfly duration later

Reply:
good point. I will transform `repeated StopPoint stop_points_nearby` in `repeated PtObject stop_points_nearby`